Mage/src/main/java/mage/game/turn/Turn.java

```diff
@@ -117,28 +119,40 @@ public boolean play(Game game, Player activePlayer) {
         this.activePlayerId = activePlayer.getId();
         resetCounts();
         game.getPlayer(activePlayer.getId()).beginTurn(game);
+        if (game.hasDayNight() && game.getTurnNum() > 1) {
```

Comment:
Is there reason to put ability's logic on such global level? It looks strange. There are multiple events like `PLAY_TURN`, `BEGINNING_PHASE`, etc

Reply:
Honestly I'm not sure exactly where it goes, but if I put it into a phase then a card like Sphinx of the Second Sun could cause issues. I'm hoping once the comprehensive rules are updated we'll get a more clear understanding of the specifics.

Reply:
Ok turns out it does go in the untap step according to the release notes from today